layouts/_default/baseof.html

```diff
@@ -58,7 +58,12 @@
       {{ end }}
     {{ end }}
     {{ block "header" . }}
-      <header data-ga-category="header">
+      <header
+        x-data="{ atTop: true }"
+        data-ga-category="header"
+        @scroll.window="atTop = window.pageYOffset>640?false:true"
+        :class="atTop ? 'relative' : 'sticky top-0 z-[1000]'"
```

Comment:
Do we need to do any tricks to keep z-[1000] from being CSS purged?

Reply:
I tested this on a draft deploy and it appears to be working without any hidden divs or tricks: https://zindex--poor-richard-spotlightpa.netlify.app/